docs/quickstart-guide.md

```diff
@@ -0,0 +1,169 @@
+
+That's it, you are up and running with Fae!
+
+# An Example CMS
+
+To give you an idea of where to go from there, here is a brief walkthrough of an example CMS setup. This example CMS will manage articles, the article categories they belong to and content on an about page.
+
+## Fae's Scaffold Generator
+
+To add an object to manage in Fae you can use Fae's built in generators to create all necessary files. They are built on the Rails generators and work very similarly.
+
+**First let's scaffold our `ArticleCategory` object:**
+
+```bash
+rails g fae:scaffold ArticleCategory name:string position:integer
+```
+
+**Next is our `Article` object:**
+
+```bash
+rails g fae:scaffold Article title:string slug:string introduction:text body:text date:date hero_image:image pdf:file article_category:references
+```
+
+This one has a couple special Fae moves in it. `image` and `file` are custom attribute types that setup everything Fae needs to manage those assets.
+
+_[Learn more about managing images and files](https://www.faecms.com/documentation/tutorials-image_and_files)_
+
+## Fae's Page Generator
+
+Fae has a built in pages module that let's you define content blocks on a page as attributes on a model. This allows you to get content for an about page via `AboutUsPage.instance` instead of dealing with hard-coded IDs.
+
+To use this module, you'll have to follow Fae's convention, but we have a custom genereator to get you started.
+
+**To generate content blocks for an `AboutUsPage`:**
+
+```bash
+rails g fae:page AboutUs hero_image:image headline:string body:text
+```
+
+_[Learn more about content blocks and pages](https://www.faecms.com/documentation/topics-pages)_
+
+**Finally let's add the scaffolded objects to the database:**
+
+```bash
+rails db:migrate
+```
+
+_Use `rake db:migrate` with Rails 4_
+
+## Customizing the Forms
+
+With those four commands we are now able to manage those three objects in our CMS. There are some things we can do now to make these forms easier to work with.
+
+### Article Categories
+
+Go to http://localhost:3000/admin/article_categories and add/update a couple categories. You'll notice that Fae picked up the `position` attribute and automatically made the categories sortable in the index view.
+
+### Articles
+
+Now let's go to create a new article at http://localhost:3000/admin/articles/new. This form already does the job, but let's make it better. Fae's generators create files in your application so you can customize this form as we please.
+
+Open `app/views/admin/articles/_form.html.slim` and focus on the `main.content` section where the form fields are:
+
+```ruby
+main.content
+  = fae_input f, :title
+  = fae_input f, :slug
+  = fae_input f, :introduction
+  = fae_input f, :body
+  = fae_input f, :date
+
+  = fae_association f, :article_category
+
+  = fae_image_form f, :hero_image
+  = fae_file_form f, :pdf
+```
+
+
+**Automatically generate the `slug` from content in the `title`.**
```

Comment:
I see your point with the inconsistency, but I don't think colons make sense here where they did above.